.ci/common-ci-settings-and-functions.sh

```diff
@@ -33,28 +33,38 @@ if [ "$PR_RUN" != "notpr" ]; then
   # in https://github.com/openshift/release/blob/master/ci-operator/config/redhat-appstudio/jvm-build-service/redhat-appstudio-jvm-build-service-main.yaml in place of the SHA
   # and OpenShift ImageStream support will fetch the correct message (i.e. it maps to a docker image tag with direct image registry references)
 
+  export JVM_BUILD_SERVICE_IMAGE_REPO JVM_BUILD_SERVICE_IMAGE_TAG
+
+  JVM_BUILD_SERVICE_IMAGE_REPO=${JVM_BUILD_SERVICE_IMAGE:-"quay.io/redhat-appstudio/hacbs-jvm-controller"}
   # for ease of use we strip the SHA, via eliminating everything starting with the '@' and whatever follows, and use the constant ImageStreamTag name instead to fetch the correct image
-  export JVM_BUILD_SERVICE_IMAGE=${JVM_BUILD_SERVICE_IMAGE%@*}
-  export JVM_BUILD_SERVICE_IMAGE_REPO=${JVM_BUILD_SERVICE_IMAGE:-"quay.io/redhat-appstudio/hacbs-jvm-controller"}
+  JVM_BUILD_SERVICE_IMAGE_REPO=${JVM_BUILD_SERVICE_IMAGE_REPO%@*}
   # Tag, i.e. ImageStreamTag, defined at: https://github.com/openshift/release/blob/master/ci-operator/config/redhat-appstudio/jvm-build-service/redhat-appstudio-jvm-build-service-main.yaml
   # as the default always reconciles in CI.  Override this for local testing.
-  export JVM_BUILD_SERVICE_IMAGE_TAG=${JVM_BUILD_SERVICE_IMAGE_TAG:-"redhat-appstudio-jvm-build-service-image"}
+  JVM_BUILD_SERVICE_IMAGE_TAG=${JVM_BUILD_SERVICE_IMAGE_TAG:-"redhat-appstudio-jvm-build-service-image"}
+
+  export JVM_BUILD_SERVICE_CACHE_IMAGE_REPO JVM_BUILD_SERVICE_CACHE_IMAGE_TAG
+
+  JVM_BUILD_SERVICE_CACHE_IMAGE_REPO=${JVM_BUILD_SERVICE_CACHE_IMAGE:-"quay.io/redhat-appstudio/hacbs-jvm-cache"}
+  JVM_BUILD_SERVICE_CACHE_IMAGE_REPO=${JVM_BUILD_SERVICE_CACHE_IMAGE_REPO%@*}
```

Comment:
I originally intended to do this change on a single line, but the shellcheck was screaming at me that I should not nest param expansions (although it worked perfectly fine in my terminal 😜  )

Reply:
shellcheck in particular / pedantic ;-)

Reply:
though I like using it when I'm not sure if I am getting too crazy with bash